qiita_pet/templates/study_base.html

```diff
@@ -20,22 +20,12 @@
   }
 
   function fill_artifact(aid) {
```

Comment:
Actually this entire function is not used anywhere (a quick grep for fill_artifact on the Qiita code only returned this occurrence). Can you remove the entire function? Thanks!